src/test/java/org/reduct/client/util/ServerClientConstants.java

```diff
@@ -1,6 +1,6 @@
 package org.reduct.client.util;
 
-public class ServerInfoConstants {
+public class ServerClientConstants {
```

Comment:
`ResponseExamples` sounds less confusing.

Reply:
Could you also check SUCCESSFUL_SERVER_INFO_RESPONSE?  The old C++ implementation used to return integers as strings, now it is normal integers. I'm wondering if it breaks something in Java.

Reply:
The ServerClientConstants renamed. In SUCCESSFUL_SERVER_INFO_RESPONSE changes all strings to integers and it was tasted successfuly.